js_modules/dagit/src/configeditor/ConfigEditorUtils.tsx

```diff
@@ -0,0 +1,183 @@
+import gql from "graphql-tag";
+import { ApolloClient } from "apollo-boost";
+import { ValidationResult } from "./codemirror-yaml/mode";
+import { ConfigEditorPipelineFragment } from "./types/ConfigEditorPipelineFragment";
+import {
+  ConfigEditorCheckConfigQuery,
+  ConfigEditorCheckConfigQueryVariables
+} from "./types/ConfigEditorCheckConfigQuery";
+import * as YAML from "yaml";
+
+export const CONFIG_EDITOR_PIPELINE_FRAGMENT = gql`
```

Comment:
Previously the Config Editor made it's own GraphQL query for this data. Now this fragment is exported by the config editor module and we incorporate it into the main pipeline query made by the execute tab. The helper methods in this file operate on this fragment and transform it into 1) the data required for the `<ConfigEditor>` and 2) the YAML scaffold. I'm not sure if there's a best practice for free-standing GraphQL fragments like this - it's not really "owned" by the ConfigEditor component anymore, it's just the data that is required for the helpers here.